index.js

```diff
@@ -1,3 +1,3 @@
 function sum(a, b) {
-    return a - b;
+    return a + b;
 }
```

Comment:
A acho que que nessa linha poderia saltar um espaço, o que acha?
```suggestion
}

```

Reply:
Concordo com o Nato!